ansible-ipi-install/roles/installer/tasks/40_create_manifest.yml

```diff
@@ -23,3 +23,4 @@
     group: "{{ ansible_user }}"
     mode: '0644'
     remote_src: yes
+  when: (release_version[0]|int >= 4) and (release_version[2]|int <= 3)
```

Comment:
shouldn't this be `release_version[0]|int >= 4) and (release_version[2]|int > 3` ? Either way theres logic here that for example in your case if you are version 5.4 it would break or in the logic I provided 5.2 it would break.